tools/interop_matrix/run_interop_matrix_tests.py

```diff
@@ -48,9 +48,8 @@
 argp.add_argument('--gcr_path',
                   default='gcr.io/grpc-testing',
                   help='Path of docker images in Google Container Registry')
-
 argp.add_argument('--release',
-                  default='master',
+                  default='all',
```

Comment:
now that you have changed the default, please update the instructions of README accordingly.

Also we have create_matrix_images.py which now has different default ('master') from this run_..._test.py.

Reply:
Done.